index.js

```diff
@@ -77,15 +77,17 @@ class Replicate {
    * @param {string} identifier - Required. The model version identifier in the format "{owner}/{name}:{version}"
    * @param {object} options
    * @param {object} options.input - Required. An object with the model inputs
-   * @param {object} [options.wait] - Whether to wait for the prediction to finish. Defaults to false
    * @param {number} [options.wait.interval] - Polling interval in milliseconds. Defaults to 250
    * @param {number} [options.wait.maxAttempts] - Maximum number of polling attempts. Defaults to no limit
```

Comment:
Why remove the docs for the boolean, but not the parameters? Are we deprecating all of the `wait` functionality?